pkg/resources/notification_integration.go

```diff
@@ -125,7 +125,7 @@ func NotificationIntegration() *schema.Resource {
 		Read:   ReadNotificationIntegration,
 		Update: UpdateNotificationIntegration,
 		Delete: DeleteNotificationIntegration,
-		Exists: NotificationIntegrationExists,
+		Check:  CheckNotificationIntegration,
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>unknown field Check in struct literal